x/datapool/client/cli/queryPool.go

```diff
@@ -120,6 +120,54 @@ func CmdGetDataValidationCertificates() *cobra.Command {
 	return cmd
 }
 
+func CmdGetDataPassRedeemReceipt() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get-data-pass-receipt [poolID] [round] [nftID]",
+		Short: "Query a data pass redeem receipt",
+		Args:  cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			queryClient := types.NewQueryClient(clientCtx)
+
+			poolID, err := strconv.ParseUint(args[0], 10, 64)
+			if err != nil {
+				return err
+			}
+
+			round, err := strconv.ParseUint(args[1], 10, 64)
+			if err != nil {
+				return err
+			}
+
+			nftID, err := strconv.ParseUint(args[2], 10, 64)
+			if err != nil {
+				return err
+			}
+
+			res, err := queryClient.DataPassRedeemReceipt(cmd.Context(), &types.QueryDataPassRedeemReceiptRequest{
+				PoolId:   poolID,
+				Round:    round,
+				NftId:    nftID,
+				Redeemer: clientCtx.GetFromAddress().String(),
```

Comment:
I think this field is not necessary.
Because usually don't get `GetFromAddress` from the query.
How about deleting this field or receiving it as a parameter?

Reply:
Or I want to know if there are other intentions

Reply:
Ah, you're right. There has no other intentions. I'll fix it. Thanks!

Reply:
Fixed in [431ac48](https://github.com/medibloc/panacea-core/pull/322/commits/431ac48367f85c89d24d20c070847bf755729664).